### ISSUE TITLE
live stream on shows the stream image but not the live strea,

### ISSUE THREAD

**Issue by guymass** [WWBN/AVideo#8388]
I see this error in Chrome console:

System error: net::ERR_TIMED_OUT
script.js?cache=1686168269_1691742412:244 parseSocketResponse {ResourceId: 1424, type: 'NEW_DISCONNECTION', callback: '', msg: {…}, uniqid: '64f8f603563cc', …}
script.js?cache=1686168269_1691742412:75 Socket onmessage NEW_DISCONNECTION {ResourceId: 1424, type: 'NEW_DISCONNECTION', callback: '', msg: {…}, uniqid: '64f8f603563cc', …}
script.js?cache=1693646162_1693646162:2663 isUserOnline NaN
?noFade=1&showCollapseButtons=1&room_users_id=3&live_transmitions_history_id=4264:678 onUserSocketConnect:error callerDisconnection is not defined
sw.js?0.44354018006034446:85 registerRoute networkFirst https://alive528.com/js/webui-popover/jquery.webui-popover.min.js
(anonymous) @ sw.js?0.44354018006034446:85
Streame:2051     GET https://alive528.com/manifest.json net::ERR_TIMED_OUT
script.js?cache=1693511719_1693542300:437 getChatTotalNew: Socket is active, we will check again in 30 seconds

I an hear the audio but not the video, this is the stream pre-image i see only:

![Screenshot 5783-12-21 at 1 00 43](https://github.com/WWBN/AVideo/assets/58882903/1b41b443-c4a2-4a2a-9fe1-8d78bb8a99a5)

**Comment by guymass** [comment]
Also in Chrome Console:

isUserOnline 4063
An unknown error occurred when fetching the script.
alive528.com/:1 Uncaught (in promise) 

![Screenshot 5783-12-21 at 1 06 49](https://github.com/WWBN/AVideo/assets/58882903/4249d471-a431-412c-b157-21ffb25edf6e)


Also on Brave:

![Screenshot 5783-12-21 at 1 04 56](https://github.com/WWBN/AVideo/assets/58882903/52cc198a-5040-4147-8314-2983415d8911)

Firefox as well:
![Screenshot 5783-12-21 at 1 00 43](https://github.com/WWBN/AVideo/assets/58882903/8a5d7378-71e6-4659-8b83-7caef829f40f)


OBS is connnected and working fine:

![Screenshot 5783-12-21 at 1 07 54](https://github.com/WWBN/AVideo/assets/58882903/0ff3e80f-4174-4ca2-a5cd-2d24395971d8)

**Comment by guymass** [comment]
On Encoder page trying to see if my stream is encoding when it finished. I see this error in chrome console:

alive528.com/:4970     GET https://live.ahava528.com/view/img/userSilhouette.jpg 404 (Not Found).

![Screenshot 5783-12-21 at 1 09 51](https://github.com/WWBN/AVideo/assets/58882903/b4c4b52a-29ea-411b-9e59-5e3a50741a18)

The file eventually was encoded correctly but still I wan't able to see the encoding process in the network encoder page.

Also trying to encode video IRLs from YouTube stopped working as well as trying to encode from FB.

**Comment by DanielnetoDotCom** [comment]
**Live Stream Quality**: It's possible that the stream being transmitted isn't in an appropriate resolution or using a compatible codec. This can sometimes result in the audio being heard without the corresponding video display. Please ensure your live stream settings match the recommended configurations for resolution and codec.

- I highly recommend checking out our guide on things to look out for when having issues with the Live Plugin: [https://github.com/WWBN/AVideo/wiki/Live-Plugin#things-to-check](https://github.com/WWBN/AVideo/wiki/Live-Plugin#things-to-check). It covers several common issues and their respective solutions.
  
- Also, ensure that you have the latest version of AVideo. Updating can often fix underlying issues as we continually optimize and rectify any known bugs.

**Comment by guymass** [comment]
OK I will check that although I never had an issue and didn't change resolution of my live stream.

Also, recently after every live stream when we open the video after encoding we see this page and only after clearing cache will it show the encoded video correctly. 

Here is a screen shot:
![Screenshot 5783-12-27 at 20 36 48](https://github.com/WWBN/AVideo/assets/58882903/95795ed9-5a66-4ee5-80a3-85e6215b7586)

And just now I made a live stream, the one you see the error above, and I only could hear the audio and no video. So I will check again the encoding, but like I said I never had an issue with my live streams encoding.

**Comment by guymass** [comment]
Makeing a new live stream I get now only audio and no video, here are some screen shots, I will post the log after:
![Screenshot 5783-12-27 at 20 50 25](https://github.com/WWBN/AVideo/assets/58882903/e4cee16c-69ce-4194-aff7-b4cd179ab852)
![Screenshot 5783-12-27 at 20 50 45](https://github.com/WWBN/AVideo/assets/58882903/e0e06b04-3bf9-4f6e-9d13-104e0d70b68f)

![Screenshot 5783-12-27 at 20 50 10](https://github.com/WWBN/AVideo/assets/58882903/66d28690-afbf-4f95-b9bb-04db7f6f638b)
![Screenshot 5783-12-27 at 20 49 55](https://github.com/WWBN/AVideo/assets/58882903/478cbdc4-4bd6-4f8b-875d-85fd308bdbc6)
![Screenshot 5783-12-27 at 20 48 55](https://github.com/WWBN/AVideo/assets/58882903/9f6e4483-23c6-40b7-bd90-ebefb7581b3a)

**Comment by guymass** [comment]
avideo.log the following constantly repeat in log:

[13-Sep-2023 20:52:51 Asia/Jerusalem] AVideoLog::DEBUG: convertVideoFileWithFFMPEG try[0]: ffmpeg  -user_agent "AVideoStreamer_fbfd7775905f11fbfe69b967368e66c8"  -i "https://alive528.com/videos/v_230912175341_v7cb6/v_230912175341_v7cb6_SD.mp4" -c:a libmp3lame "/var/www/html/AVideo/videos/v_230912175341_v7cb6/v_230912175341_v7cb6.mp3" 1> '/var/www/html/AVideo/videos/v_230912175341_v7cb6/v_230912175341_v7cb6.mp3.log' 2>&1 SCRIPT_NAME: /view/videosList.php
[13-Sep-2023 20:52:51 Asia/Jerusalem] AVideoLog::DEBUG: convertVideoFileWithFFMPEG try[0] output: [] SCRIPT_NAME: /view/videosList.php
[13-Sep-2023 20:52:51 Asia/Jerusalem] AVideoLog::DEBUG: convertVideoFileWithFFMPEG: age: 1694627571 too long without change, unlock it https://alive528.com/videos/v_230912175341_v7cb6/v_230912175341_v7cb6_SD.mp4 SCRIPT_NAME: /plugin/PlayerSkins/mediaSession.json.php
[13-Sep-2023 20:52:51 Asia/Jerusalem] AVideoLog::DEBUG: get_ffmpeg default ffmpeg  -user_agent "AVideoStreamer_fbfd7775905f11fbfe69b967368e66c8"  SCRIPT_NAME: /plugin/PlayerSkins/mediaSession.json.php
[13-Sep-2023 20:52:51 Asia/Jerusalem] AVideoLog::DEBUG: convertVideoFileWithFFMPEG try[0]: ffmpeg  -user_agent "AVideoStreamer_fbfd7775905f11fbfe69b967368e66c8"  -i "https://alive528.com/videos/v_230912175341_v7cb6/v_230912175341_v7cb6_SD.mp4" -c:a libmp3lame "/var/www/html/AVideo/videos/v_230912175341_v7cb6/v_230912175341_v7cb6.mp3" 1> '/var/www/html/AVideo/videos/v_230912175341_v7cb6/v_230912175341_v7cb6.mp3.log' 2>&1 SCRIPT_NAME: /plugin/PlayerSkins/mediaSession.json.php
[13-Sep-2023 20:52:51 Asia/Jerusalem] AVideoLog::DEBUG: convertVideoFileWithFFMPEG try[0] output: [] SCRIPT_NAME: /plugin/PlayerSkins/mediaSession.json.php
[13-Sep-2023 20:52:51 Asia/Jerusalem] AVideoLog::DEBUG: CustomizeUser::canShareVideosFromVideo canShareVideosFromUser(2) = false SCRIPT_NAME: /view/modeYoutube.php
[13-Sep-2023 20:52:51 Asia/Jerusalem] AVideoLog::DEBUG: convertVideoFileWithFFMPEG: age: 1694627571 too long without change, unlock it https://alive528.com/videos/v_230912175341_v7cb6/v_230912175341_v7cb6_SD.mp4 SCRIPT_NAME: /view/videosList.php
[13-Sep-2023 20:52:51 Asia/Jerusalem] AVideoLog::DEBUG: get_ffmpeg default ffmpeg  -user_agent "AVideoStreamer_fbfd7775905f11fbfe69b967368e66c8"  SCRIPT_NAME: /view/videosList.php

**Comment by guymass** [comment]
[13-Sep-2023 20:52:52 Asia/Jerusalem] AVideoLog::DEBUG: getCurrentPage current>1000 ERROR [1897] [{"file":"\/var\/www\/html\/AVideo\/objects\/bootGrid.php","line":50,"function":"getCurrentPage"},{"file":"\/var\/www\/html\/AVideo\/objects\/video.php","line":1758,"function":"getSqlFromPost","class":"BootGrid","type":"::"},{"file":"\/var\/www\/html\/AVideo\/plugin\/Gallery\/functions.php","line":81,"function":"getAllVideos","class":"Video","type":"::"},{"file":"\/var\/www\/html\/AVideo\/plugin\/Gallery\/view\/mainArea.php","line":137,"function":"createGallery"},{"file":"\/var\/www\/html\/AVideo\/plugin\/Gallery\/view\/modeGallery.php","line":21,"args":["\/var\/www\/html\/AVideo\/plugin\/Gallery\/view\/mainArea.php"],"function":"include"},{"file":"\/var\/www\/html\/AVideo\/view\/index_firstPage.php","line":58,"args":["\/var\/www\/html\/AVideo\/plugin\/Gallery\/view\/modeGallery.php"],"function":"require"},{"file":"\/var\/www\/html\/AVideo\/view\/index.php","line":20,"args":["\/var\/www\/html\/AVideo\/view\/index_firstPage.php"],"function":"include"}] SCRIPT_NAME: /view/index.php
[13-Sep-2023 20:52:52 Asia/Jerusalem] AVideoLog::DEBUG: getCurrentPage current>1000 ERROR NOT LOGGED die [1897] https://alive528.com/view/?page=2410&page=2420&tags_id=30&lang=cs {"REDIRECT_SCRIPT_URL":"\/page\/2410","REDIRECT_SCRIPT_URI":"https:\/\/alive528.com\/page\/2410","REDIRECT_proto":"https","REDIRECT_HTTPS":"on","REDIRECT_SSL_TLS_SNI":"alive528.com","REDIRECT_STATUS":"200","SCRIPT_URL":"\/page\/2410","SCRIPT_URI":"https:\/\/alive528.com\/page\/2410","proto":"https","HTTPS":"on","SSL_TLS_SNI":"alive528.com","HTTP_HOST":"alive528.com","HTTP_AMP_CACHE_TRANSFORM":"google;v=\"1..8\"","HTTP_CONNECTION":"keep-alive","HTTP_ACCEPT":"text\/html,application\/xhtml+xml,application\/signed-exchange;v=b3,application\/xml;q=0.9,*\/*;q=0.8","HTTP_FROM":"googlebot(at)googlebot.com","HTTP_USER_AGENT":"Mozilla\/5.0 (Linux; Android 6.0.1; Nexus 5X Build\/MMB29P) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/116.0.5845.179 Mobile Safari\/537.36 (compatible; Googlebot\/2.1; +http:\/\/www.google.com\/bot.html)","HTTP_ACCEPT_ENCODING":"gzip, deflate, br","PATH":"\/usr\/local\/sbin:\/usr\/local\/bin:\/usr\/sbin:\/usr\/bin:\/sbin:\/bin:\/snap\/bin","SERVER_SIGNATURE":"<address>Apache\/2.4.52 (Ubuntu) Server at alive528.com Port 443<\/address>\n","SERVER_SOFTWARE":"Apache\/2.4.52 (Ubuntu)","SERVER_NAME":"alive528.com","SERVER_ADDR":"178.162.220.42","SERVER_PORT":"443","REMOTE_ADDR":"66.249.76.41","DOCUMENT_ROOT":"\/var\/www\/html\/AVideo","REQUEST_SCHEME":"https","CONTEXT_PREFIX":"","CONTEXT_DOCUMENT_ROOT":"\/var\/www\/html\/AVideo","SERVER_ADMIN":"admin@avaha528.com","SCRIPT_FILENAME":"\/var\/www\/html\/AVideo\/view\/index.php","REMOTE_PORT":"50969","REDIRECT_URL":"\/page\/2410","REDIRECT_QUERY_STRING":"page=2410&page=2420&tags_id=30&lang=cs","GATEWAY_INTERFACE":"CGI\/1.1","SERVER_PROTOCOL":"HTTP\/1.1","REQUEST_METHOD":"GET","QUERY_STRING":"page=2410&page=2420&tags_id=30&lang=cs","REQUEST_URI":"\/page\/2410?page=2420&tags_id=30&lang=cs","SCRIPT_NAME":"\/view\/index.php","PHP_SELF":"\/view\/index.php","REQUEST_TIME_FLOAT":1694627572.657974,"REQUEST_TIME":1694627572} SCRIPT_NAME: /view/index.php

**Comment by guymass** [comment]
Last one might be clearer, but I see few errors and idk what they relate to.

[13-Sep-2023 20:57:00 Asia/Jerusalem] AVideoLog::DEBUG: getCurrentPage current>1000 ERROR [1897] [{"file":"\/var\/www\/html\/AVideo\/objects\/bootGrid.php","line":50,"function":"getCurrentPage"},{"file":"\/var\/www\/html\/AVideo\/objects\/video.php","line":1758,"function":"getSqlFromPost","class":"BootGrid","type":"::"},{"file":"\/var\/www\/html\/AVideo\/plugin\/Gallery\/functions.php","line":81,"function":"getAllVideos","class":"Video","type":"::"},{"file":"\/var\/www\/html\/AVideo\/plugin\/Gallery\/view\/mainArea.php","line":137,"function":"createGallery"},{"file":"\/var\/www\/html\/AVideo\/plugin\/Gallery\/view\/modeGallery.php","line":21,"args":["\/var\/www\/html\/AVideo\/plugin\/Gallery\/view\/mainArea.php"],"function":"include"},{"file":"\/var\/www\/html\/AVideo\/view\/index_firstPage.php","line":58,"args":["\/var\/www\/html\/AVideo\/plugin\/Gallery\/view\/modeGallery.php"],"function":"require"},{"file":"\/var\/www\/html\/AVideo\/view\/index.php","line":20,"args":["\/var\/www\/html\/AVideo\/view\/index_firstPage.php"],"function":"include"}] SCRIPT_NAME: /view/index.php
[13-Sep-2023 20:57:00 Asia/Jerusalem] AVideoLog::DEBUG: getCurrentPage current>1000 ERROR NOT LOGGED die [1897] https://alive528.com/view/?page=1964&page=2418&lang=th {"REDIRECT_SCRIPT_URL":"\/page\/1964","REDIRECT_SCRIPT_URI":"https:\/\/alive528.com\/page\/1964","REDIRECT_proto":"https","REDIRECT_HTTPS":"on","REDIRECT_SSL_TLS_SNI":"alive528.com","REDIRECT_STATUS":"200","SCRIPT_URL":"\/page\/1964","SCRIPT_URI":"https:\/\/alive528.com\/page\/1964","proto":"https","HTTPS":"on","SSL_TLS_SNI":"alive528.com","HTTP_HOST":"alive528.com","HTTP_AMP_CACHE_TRANSFORM":"google;v=\"1..8\"","HTTP_CONNECTION":"keep-alive","HTTP_ACCEPT":"text\/html,application\/xhtml+xml,application\/signed-exchange;v=b3,application\/xml;q=0.9,*\/*;q=0.8","HTTP_FROM":"googlebot(at)googlebot.com","HTTP_USER_AGENT":"Mozilla\/5.0 (Linux; Android 6.0.1; Nexus 5X Build\/MMB29P) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/116.0.5845.179 Mobile Safari\/537.36 (compatible; Googlebot\/2.1; +http:\/\/www.google.com\/bot.html)","HTTP_ACCEPT_ENCODING":"gzip, deflate, br","PATH":"\/usr\/local\/sbin:\/usr\/local\/bin:\/usr\/sbin:\/usr\/bin:\/sbin:\/bin:\/snap\/bin","SERVER_SIGNATURE":"<address>Apache\/2.4.52 (Ubuntu) Server at alive528.com Port 443<\/address>\n","SERVER_SOFTWARE":"Apache\/2.4.52 (Ubuntu)","SERVER_NAME":"alive528.com","SERVER_ADDR":"178.162.220.42","SERVER_PORT":"443","REMOTE_ADDR":"66.249.64.72","DOCUMENT_ROOT":"\/var\/www\/html\/AVideo","REQUEST_SCHEME":"https","CONTEXT_PREFIX":"","CONTEXT_DOCUMENT_ROOT":"\/var\/www\/html\/AVideo","SERVER_ADMIN":"admin@avaha528.com","SCRIPT_FILENAME":"\/var\/www\/html\/AVideo\/view\/index.php","REMOTE_PORT":"52039","REDIRECT_URL":"\/page\/1964","REDIRECT_QUERY_STRING":"page=1964&page=2418&lang=th","GATEWAY_INTERFACE":"CGI\/1.1","SERVER_PROTOCOL":"HTTP\/1.1","REQUEST_METHOD":"GET","QUERY_STRING":"page=1964&page=2418&lang=th","REQUEST_URI":"\/page\/1964?page=2418&lang=th","SCRIPT_NAME":"\/view\/index.php","PHP_SELF":"\/view\/index.php","REQUEST_TIME_FLOAT":1694627820.369389,"REQUEST_TIME":1694627820} SCRIPT_NAME: /view/index.php
[13-Sep-2023 20:57:00 Asia/Jerusalem] AVideoLog::DEBUG: contentSearchFound NOT FOUND [{"file":"\/var\/www\/html\/AVideo\/plugin\/Gallery\/view\/modeGallery.php","line":21,"function":"include"},{"file":"\/var\/www\/html\/AVideo\/view\/index_firstPage.php","line":58,"args":["\/var\/www\/html\/AVideo\/plugin\/Gallery\/view\/modeGallery.php"],"function":"require"},{"file":"\/var\/www\/html\/AVideo\/view\/index.php","line":20,"args":["\/var\/www\/html\/AVideo\/view\/index_firstPage.php"],"function":"include"}] SCRIPT_NAME: /view/index.php

**Comment by DanielnetoDotCom** [comment]
I've noticed several errors in your logs, particularly those labeled as "Uncaught Error." From my assessment, it appears that none of these errors are directly related to audio issues during live streaming.

I've already shared my thoughts on the matter in a previous comment within the [audio-only issue thread](https://github.com/WWBN/AVideo/issues/8388#issuecomment-1710111520). 

However, it seems there might be some outdated aspects in your setup that are making it challenging for me to assist you effectively. To facilitate smoother troubleshooting and resolution, I recommend opening separate issue threads for each error or concern that you encounter.

 This way, we can address each issue individually and ensure that your system is up to date and fully functional.

**Comment by guymass** [comment]
Sorry for late reply couldn't be online for a while. I have done what was recommended for the OBS setting and lowered it and now the video shows again and encoding seems to work for that file. I will be able to run more test this week and see if its all been resolved.
